README.md

```diff
@@ -140,3 +140,28 @@ secrets.
           twitter_consumer_secret: ${{ secrets.TWITTER_CONSUMER_API_SECRET }}
           twitter_consumer_key: ${{ secrets.TWITTER_CONSUMER_API_KEY }}
 ```
+
+### Deploy to Mastodon
+
+To deploy to Mastodon (in addiction to Slack) you are required to set `deploy_mastodon`
```

Comment:
```suggestion
To deploy to Mastodon (in addition to Slack) you are required to set `deploy_mastodon`
```
Is slack + mastodon required? if not, we can remove the "in addition" statement.

Reply:
Actually, you're right.  I should remove the "in addition" statement.  Will do.